firestore/firestore_snippets/query.go

```diff
@@ -249,6 +249,16 @@ func createMultipleStartAtQuery(client *firestore.Client) {
 	// [END fs_start_at_multiple]
 }
 
+func createInQuery(ctx context.Context, client *firestore.Client) error {
+	cities := client.Collection("cities")
+	// [START fs_in_query]
```

Comment:
```suggestion
	// [START fs_query_filter_in]
```

Reply:
Can you also move this tag start above `cities := ...` We want that ref in the snippet.